website/src/views/pageStateHandlers/toLapisFilterWithoutVariant.ts

```diff
@@ -1,12 +1,12 @@
 import type { LapisFilter } from '@genspectrum/dashboard-components/util';
 
-import type { Dataset } from '../View.ts';
+import type { DatasetFilter } from '../View.ts';
 
 export function toLapisFilterWithoutVariant(
-    pageState: Dataset,
+    datasetFilter: DatasetFilter,
```

Comment:
This change still needs to be made on the GenericAnalyzeSingleVariantPage I think? I am looking at all the parts of the code where toLapisFilterWithoutVariant is used

Reply:
Where exactly do you mean? I didn't find it

(There is
```
    public toLapisFilterWithoutVariant(pageState: DatasetAndVariantData): LapisFilter {
        return toLapisFilterWithoutVariant(pageState.datasetFilter, this.constants.additionalFilters);
    }
```
in case you missed. It that too confusing? Should I revert this change?)